public/app/percona/settings/Settings.service.ts

```diff
@@ -60,4 +60,5 @@ const toModel = (response: SettingsPayload): Settings => ({
     frequentInterval: response.stt_check_intervals.frequent_interval,
   },
   backupEnabled: response.backup_management_enabled,
+  isConnectedToPortal: response.connected_to_platform,
```

Comment:
@jprukner What is driving this change? We cannot connect to the Platform because Platform is not software. However, the Portal is software so connecting to it seems a real thing. I couldn't get more info from the ticket, because it talks about the feature.

Reply:
This was requested by @palash25. https://github.com/percona/pmm/pull/792#discussion_r758361113

Reply:
https://github.com/percona/pmm/pull/792#discussion_r761893516